packages/expo-video/android/src/main/java/expo/modules/video/FullscreenPlayerActivity.kt

```diff
@@ -49,6 +49,7 @@ class FullscreenPlayerActivity : Activity() {
     if (Build.VERSION.SDK_INT >= 34) {
```

Comment:
Looks like ifs for `Build.VERSION.SDK_INT` are not consistent – sometimes we compare it to a number and sometimes to predefined `Build.VERSION_CODES`. Maybe we could unify it in this PR?